mlflow/gateway/config.py

```diff
@@ -160,11 +161,32 @@ class MlflowModelServingConfig(ConfigModel):
     model_server_url: str
 
 
+class AWSBaseConfig(pydantic.BaseModel):
+    aws_region: Optional[str] = None
+
+
+class AWSRole(AWSBaseConfig):
+    aws_role_arn: str
+    session_length_seconds: int = 15 * 60
```

Comment:
How important is it to provide a default value for session length that is shorter than the default provided by ARN registration through AWS? The default is 1 hour per the AWS docs. 
If we set the default to be the same as their service, that seems ok (if it's not something that people generally override), but since the default here is shorter than that, it seems like this should be admin-controlled during route configuration.

Reply:
@BenWilson2 Not terribly.  It's weakly important that it be configured to be bit shorter than whatever the AWS-configured maximum duration is, so that a request arriving right at expiry doesn't race `AWSBedrockProvider._client_expired` against the actual expiration.  A much shorter default than theirs is a strong guarantee of that condition :P.  

I can add some docs to that effect.

Reply:
Adding this info in docs seems good to me!

Reply:
(Don't worry about this for this PR - I'll file a follow-up PR to add this to the docs)